### PR TITLE
Add new plots to ECAL Offline DQM with Pedestal and Laser transparency correction from the Database

### PR DESCRIPTION
**Pull request by abhih1**
#### PR description:

At the request of the ECAL DPG, new plots are added to the Offline DQM workflow, involving pedestal RMS values and laser transparency correction of crystals, read from the database directly, as functions of eta.
These plots would be useful in general for Offline DQM and also crucial in understanding if the conditions loaded by CMSSW changes from one run to the other in RelVal MC validation.

#### PR validation:

The code changes were validated by running the DQM relval workflow 136.874 using the runTheMatrix script
`runTheMatrix.py -l 136.874 --ibeos`

The resultant DQM output file was examined by uploading it to an Offline DQM test GUI and confirming the new plots.
The plots were also approved by the ECAL DPG team.

#### Backport:
A backport PR to CMSSW_11_3_X has been created here: https://github.com/cms-sw/cmssw/pull/34341

### CODE REVIEW

File: DQM/EcalMonitorTasks/src/OccupancyTask.cc [hunk 1]
```diff
@@ -25,6 +24,11 @@ namespace ecaldqm {
     return false;
   }
 
+  void OccupancyTask::beginRun(edm::Run const&, edm::EventSetup const& _es) {
+    _es.get<EcalLaserDbRecord>().get(laser);
```
Comment (sroychow) [inline DQM/EcalMonitorTasks/src/OccupancyTask.cc:28]: This access of data from ES without token is deprecated. Please follow the instructions in this [twiki](https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideHowToGetDataFromES#In_ED_module) to update accessing the data from  ES.

File: DQM/EcalMonitorTasks/src/PresampleTask.cc [hunk 2]
```diff
@@ -28,6 +29,11 @@ namespace ecaldqm {
     return false;
   }
 
+  void PresampleTask::beginRun(edm::Run const&, edm::EventSetup const& _es) {
+    _es.get<EcalPedestalsRcd>().get(pPeds);
```
Comment (sroychow) [inline DQM/EcalMonitorTasks/src/PresampleTask.cc:33]: Please see the previous comment on ES data access.

File: DQM/EcalMonitorTasks/src/OccupancyTask.cc (hunk 1 above)
Comment (abhih1) [reply DQM/EcalMonitorTasks/src/OccupancyTask.cc]: Thanks. It is updated now.

**Comment by sroychow** [comment]
@cmsbuild  please test

**Comment by jfernan2** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**